### ISSUE TITLE
asm reopen err

### ISSUE THREAD

**Issue by Shane-XB-Qian** [vim/vim#15035]
### Steps to reproduce

1. vim --clean
2. vsp foo.s
3. close
4. vsp foo.s
5. err:
```
E488: Trailing characters: < indentkeys<
```
the undo_ftplugin:
`setl commentstring< comments< include< | unlet! b:match_ignorecase b:match_words b:match_skipindentexpr< indentkeys<`
somehow appended `indentexpr< indentkeys<` and no space.

### Expected behaviour

no err.
// and not sure where `indentexpr< indentkeys<` came from.

### Version of Vim

v9.1.494

### Environment

linux

**Comment by chrisbra** [comment]
hm, cannot reproduce. Do you still have a `indent/asm.vim` indent plugin? I think the error you notice was fixed by 98b12ede31754071f36fb7a733244 but later then the indent plugin was removed totally: 76174e71101503900d54d38e00b3a

**Comment by Shane-XB-Qian** [comment]
> hm, cannot reproduce. Do you still have a `indent/asm.vim` indent plugin? I think the error you notice was fixed by 98b12ede31754071f36fb7a733244 but later then the indent plugin was removed totally: 76174e71101503900d54d38e00b3a

```
 27: /usr/local/share/vim/vim91/ftplugin/asm.vim
 28: /usr/local/share/vim/vim91/indent/asm.vim
 29: /usr/local/share/vim/vim91/syntax/asm.vim
```

yes, but i did `make distclean` then install, um.............

-- 
shane.xb.qian

**Comment by Shane-XB-Qian** [comment]
thanks, let it be.
